bin/setup-production

```diff
@@ -2,7 +2,11 @@
 
 require 'pathname'
 require 'fileutils'
-require 'dotenv'
+
+# always run this in production environment
+ENV['RAILS_ENV'] = 'production'
+
+require_relative '../config/boot'
```

Comment:
Should this be called before the `require` statements above? Or is it *ok* here since the above `require` statements aren't gems (since you removed the `dotenv`).

Reply:
require_relative has to be done after ENV['RAILS_ENV'] is set. the other requires don't matter but I could probably move them down with the require_relative...